_common.sh

```diff
@@ -190,7 +190,12 @@ function make_temp_directory {
 
 	mkdir -p "${TEMP_DIR}"
 
-	cp -r "${1}"/* "${TEMP_DIR}"
+	local resource_dir
+
+	for resource_dir in "${@}"
+	do
+        cp -r "${resource_dir}"/* "${TEMP_DIR}"
```

Comment:
@natocesarrego no it will not copy the Dockerfile because of the /* part. the parameters would look like `cp -r templates/jdk11 templates/jdk-common/* "${TEMP_DIR}"` so the /* will be missing for the first parameter.